### DISCUSSION TITLE
Innspill til informasjonsmodell

### DISCUSSION THREAD

**Discussion by eirikbroen** [NorskHelsenett/Tillitsrammeverk#24] [General]
Innspill til hvordan informasjonsmodell kan fungere i kommunal pleie og omsorg

| Informasjon           | Beskrivelse                                                                                                  | Informasjonskilde | Påkrevd | Eksempelverdi                                  
|-----------------------|--------------------------------------------------------------------------------------------------------------|-------------------|---------|------------------------------------------------|
| "pid"                 | Fødselsnummer fra folkeregisteret                                                                            | HelseID           | **Ja**  | 20026206651                                    |
| "structural_role"     | Indikerer hvorvidt dette er et helsepersonell med/uten lisens/autorisasjon eller person med sekretærfunksjon | Konsumentens EPJ  | **Nei** ||
| "hpr_nr"              | Helsepersonellets HPR-nummer, dersom det finnes                                                              | HelseID           | **Nei** | 7089678                                        |
| "authorization"       | Helsepersonellets autorissjoner, dersom de finnes                                                            | HelseID           | **Nei** | Lege (Volven 9060)                                          |
| "licence"             | Helsepersonellets lisenser, dersom de finnes                                                                 | HelseID           | **Nei** ||
| "functional_role"     | Helsepersonellets funksjonelle rolle hos virksomheten                                                        | Konsumentens EPJ  | **Nei**  | Lege                              |
| "clinical_speciality" | Helsepersonellets kliniske spesialitet                                                                       | Konsumentens EPJ  | **Nei** ||
| "legal_entity_id"     | Den dataansvarlige virksomhetens org.nr                                                                      | Konsumentens EPJ  | **Ja**  | 997506499                                      |
| "legal_entity_name"   | Den dataansvarlige virksomhetens navn                                                                        | Konsumentens EPJ  | **Ja**  | OSLO KOMMUNE HELSEETATEN                       |
| "point_of_care_id"    | Behandlingsstedets org.nr., dersom det er relevant                                                           | Konsumentens EPJ  | **Nei** | 875300342                                      |
| "point_of_care_name"  | Behandlingsstedets navn, dersom det er relevant                                                              | Konsumentens EPJ  | **Nei** | MADSERUDHJEMMET                                |
| "facility_type"       | Helsetjenestetyper som leveres ved virksomheten                                                              | Konsumentens EPJ  | **Ja**  | KP01 - Legetjeneste ved sykehjem (Volven 8663) |
| "locality"            | Avdeling/org.enhet hvor helsepersonellet yter helsehjelp                                                     | Konsumentens EPJ  | **Nei** ||
| "purpose_of_use"      | Helsepersonellets formåle med helseopplysningene (til hva de skal brukes)                                    | Konsumentens EPJ  | **Ja**  | 08 - Avlastning i institusjon (Volven 9151)    |
| "patient_id"          | Unik identifikator for pasienten                                                                             | Konsumentens EPJ  | **Ja**  | 13116900216                                    |

Mulige verdier i kommunal pleie og omsorg for _facility_type_  (helsehjelpstjeneste)

| Verdi                             | Kodeverk |
|-----------------------------------|----------|
| KP01 Legetjeneste ved sykehjem mv | [8663](https://volven.no/produkt.asp?id=494006&catID=3&subID=8)     |
| KP02 Sykepleietjeneste            | [8663](https://volven.no/produkt.asp?id=494006&catID=3&subID=8)     | 
| KD0501 Saksbehandling             | [8663](https://volven.no/produkt.asp?id=494006&catID=3&subID=8)     | 
| KP02 Sykepleietjeneste            | [8663](https://volven.no/produkt.asp?id=494006&catID=3&subID=8)     | 
| KX06 Kreftkoordinator             | [8663](https://volven.no/produkt.asp?id=494006&catID=3&subID=8)     | 
| KX07 Demenskoordinator            | [8663](https://volven.no/produkt.asp?id=494006&catID=3&subID=8)     | 
| 10 Rehabilitering                 | [8666](https://volven.no/produkt.asp?id=494008&catID=3&subID=8)     | 
| A02 Ergoterapi                    | [8666](https://volven.no/produkt.asp?id=494008&catID=3&subID=8)     | 
| A03 Fysioterapi                   | [8666](https://volven.no/produkt.asp?id=494008&catID=3&subID=8)     | 
| KP03 Psykisk helsetjeneste        | [8666](https://volven.no/produkt.asp?id=494008&catID=3&subID=8)     | 
| KP05 Psykologtjeneste             | [8666](https://volven.no/produkt.asp?id=494008&catID=3&subID=8)     | 
| R Rus og avhengighet              | [8666](https://volven.no/produkt.asp?id=494008&catID=3&subID=8)     |

Mulige verdier for å beskrive tjenstlig behov _purpose_of_use i kommunal pleie og omsorg

| Verdi                                                  | Kodeverk                                                        |
|--------------------------------------------------------|-----------------------------------------------------------------|
| 07 Avlastning – utenfor institusjon                    | [9151](https://volven.no/produkt.asp?id=493180&catID=3&subID=8) |
| 08 Avlastning – i institusjon                          |[9151](https://volven.no/produkt.asp?id=493180&catID=3&subID=8)|
| 08.1 Avlastning i institusjon – rullerende             |[9151](https://volven.no/produkt.asp?id=493180&catID=3&subID=8)|
| 08.2 Avlastning i institusjon – ikke rullerende        |[9151](https://volven.no/produkt.asp?id=493180&catID=3&subID=8)|
| 15 Helsetjenester i hjemmet                            |[9151](https://volven.no/produkt.asp?id=493180&catID=3&subID=8)|
| 17 Dagopphold i institusjon                            |[9151](https://volven.no/produkt.asp?id=493180&catID=3&subID=8)| 
| 18 Tidsbegrenset opphold – utredning/behandling        |[9151](https://volven.no/produkt.asp?id=493180&catID=3&subID=8)|
| 19 Tidsbegrenset opphold – habilitering/rehabilitering |[9151](https://volven.no/produkt.asp?id=493180&catID=3&subID=8)| 
| 20 Tidsbegrenset opphold – annet                       |[9151](https://volven.no/produkt.asp?id=493180&catID=3&subID=8)|
|  21 Langtidsopphold i institusjon                      |[9151](https://volven.no/produkt.asp?id=493180&catID=3&subID=8)|
| 22 Nattopphold i institusjon                           |[9151](https://volven.no/produkt.asp?id=493180&catID=3&subID=8)|

**Comment by OK-SverreM** [comment]
Angående "functional_role", så tror jeg ikke vi skal legge opp til duplisert bruk av autorisasjonsrollekodene (Volven, OID 9060) her.

I tidligere runder med både HSØ og PDS har det vært snakk om tre distinkte typer roller:

1. **Autorisasjonsrolle** (OID 9060). Denne er grei, hentes fra HPR.
2. **Ansattrolle**. I betydningen "stilling du er ansatt i". For denne type mer administrativ informasjon, vil primærkilden være HR-systemet (_ikke_ EPJ). Her ser jeg det kan gi mening å benytte STYRK kodeverk, som vel nettopp benyttes for innrapportering fra HR-system til arbeidsgiver/arbeidstaker-registeret (via EDAG-ordningen). Men: Vår konklusjon så langt er at denne informasjonen gir begrenset / ingen merverdi utover autorisasjonsrollen, + at STYRK-koder ikke finnes i EPJ-systemene (det er ikke deres naturlige habitat).
3. **Funksjonell rolle**. HSØ har gjort et omfattende arbeid med å etablere et internt system for dette (se https://github.com/NorskHelsenett/Tillitsrammeverk/blob/main/specs/misc/sammenligning_av_funksjonell_rolle.md). Men det finnes ikke noe tilsvarende for kommunene f.eks. 

[@richardhus ](https://github.com/richardhus) Kjenner du deg igjen i det jeg har skrevet i punktene over?

Jeg tror vi gjør klokt i å ikke blande disse ulike typene roller sammen med hverandre. 
Hvis vi skal ha et rollebegrep i tillegg til **Autorisasjonsrolle**, så bør det være veldig klart om vi snakker om **Ansattrolle** (hvor STYRK er aktuelt som kodeverk) eller det mer helsetjenestespesifikke **Funksjonell rolle** (hvor det per nå ikke finnes ? / er pekt på noe felles kodeverk). 

Konklusjon på kort sikt (dvs. 2023 / realiseringen av behovskøelement # 27)?
Hvis man skal ta med Ansattrolle og/eller Funksjonell rolle i Tillitsrammeverket, så kan ikke disse per nå settes som obligatoriske.
I tillegg må det være helt tydelig hvilken av de to rolletypene man tar med, og for hvilket formål. Det tjenstlige behovet (som vel skal være det viktigste grunnlaget for tilgang i Tillitsmodellen) belyses neppe så mye bedre ved å ha med en rekke forskjellige typer roller?

**Reply by stvgr** [reply]
Stavanger er enige i dette

**Comment by eirikbroen** [comment]
Jeg oppdaterer tabell i starten av denne diskusjonstråden slik at den reflekterer siste forslag fra @OK-SverreM her
